### DISCUSSION TITLE
Appearence Theme Preferences is not the same in public view

### DISCUSSION THREAD

**Discussion by rafael-lucas-rlns** [community/community#19148] [Other Feature Feedback, Questions, & Ideas]
I tested access in an anonymous window, and the settings I applied were not available to public access.

**Comment by itsgareth** [comment]
@rafael-lucas-rlns The appearance settings on Github are just designed to `Choose how GitHub looks to you` there is currently no way to set a default theme for everybody that views your profile.

**Reply by rafael-lucas-rlns** [reply]
> @rafael-lucas-rlns The appearance settings on Github are just designed to `Choose how GitHub looks to you` there is currently no way to set a default theme for everybody that views your profile.

yes, i understand this, but I refer to the public profile (of a person not logged in). Shouldn't it get the user's preferred settings instead of the github default?

**Reply by itsgareth** [reply]
as far as i am aware, github has always used light mode if a user is not logged in and there is no way to change this.
